packages/inter-protocol/src/price/fluxAggregator.contract.js

```diff
@@ -0,0 +1,92 @@
+import { AssetKind, makeIssuerKit } from '@agoric/ertp';
+import { handleParamGovernance } from '@agoric/governance';
+import { assertAllDefined, makeTracer } from '@agoric/internal';
+import { E } from '@endo/eventual-send';
+import { provideFluxAggregator } from './fluxAggregator.js';
+
+const trace = makeTracer('FluxAgg');
```

Comment:
```suggestion
const trace = makeTracer('FluxAgg', false);
```

Reply:
I'm going to leave the debugging in until this is R4R